modules/nodes-cluster-worker-latency-profiles-using.adoc

```diff
@@ -138,3 +139,80 @@ $ oc get KubeControllerManager -o yaml | grep -i workerlatency -A 5 -B 5
 
 To change the medium profile to default or change the default to medium, edit the `node.config` object and set the `spec.workerLatencyProfile` parameter to the appropriate value.
 
+[id="nodes-cluster-worker-latency-profiles-using-at-creation_{context}"]
+== Using worker latency profiles at cluster creation
+
+To implement a worker latency profile in the manifest, follow these steps:
+
+* Create the manifest needed to build the cluster.
+* Edit the `node.config` object to add the name of the profile. You can change the profile at any time as latency increases or decreases.
+* Define `workerLatencyProfile` using the `"Default"` profile.
++
+[source,terminal]
+----
+$ cat <cluster-install-dir>/manifests/config-node-default-profile.yaml
+----
++
+.Example output
+[source,terminal]
+----
+apiVersion: config.openshift.io/v1
+kind: Node
+metadata:
+ name: cluster
+spec:
+ workerLatencyProfile: "Default"
+----
+
+[id="nodes-cluster-worker-latency-profiles-examining_{context}"]
+== Example steps for displaying resulting values of 'workerLatencyProfile'
+
+You can display the values in the `workerLatencyProfile` with the following commands.
+
+
+* Check the `default-not-ready-toleration-seconds` and `default-unreachable-toleration-seconds` of the Kube API Server:
++
+[source,terminal]
+----
+$ oc get KubeAPIServer -o yaml | grep -A 1 default-
+----
++
+.Example output
+[source,terminal]
+----
+default-not-ready-toleration-seconds:
+- "300"
+default-unreachable-toleration-seconds:
+- "300"
+----
+
+* Check the values of the `node-monitor-grace-period` from the Kube Controller Manager:
++
+[source,terminal]
+----
+$ oc get KubeControllerManager -o yaml | grep -A 1 node-monitor
+----
++
+.Example output
+[source,terminal]
+----
+node-monitor-grace-period:
+- 40s
+----
+
+* The `nodeStatusUpdateFrequency` value from the Kubelet:
++
+[source,terminal]
+----
+$ oc debug node/<worker-node-name>
+$ chroot /host
```

Comment:
This needs to be another step. Why are we chrooting? Is this a verification step? If so, start a new numbered step section under `.Verification`.

Reply:
I'm asking about this, no conclusion yet. Will leave this conversation open.

Reply:
I explained the "chroot /host". The command and that usage is common in the documentation and I added a couple sentences to explain the process. Those three commands only provide the result when run together in the sequence shown, so none of the three "stands alone".